app/src/main/java/org/fossasia/susi/ai/MainApplication.java

```diff
@@ -45,5 +48,34 @@ public void onCreate() {
         Realm.init(this);
         RealmConfiguration config = new RealmConfiguration.Builder().deleteRealmIfMigrationNeeded().build();
         Realm.setDefaultConfiguration(config);
+
+        if(BuildConfig.DEBUG) {
+            Timber.plant(new Timber.DebugTree(){
+                //Add the line number to the tag
+                @Override
+                protected String createStackElementTag(StackTraceElement element){
+                    return super.createStackElementTag(element) + ": " + element.getLineNumber();
+
+                }
+            });
+        }else{
+            //Release mode
+            Timber.plant(new ReleaseLogTree());
+        }
+    }
+
+    private static class ReleaseLogTree extends Timber.Tree {
+
+        @Override
+        protected void log(int priority, String tag, @NonNull String message, Throwable throwable) {
+            if (priority == Log.DEBUG || priority == Log.VERBOSE || priority == Log.INFO) {
+                return;
+            }
+
+            if (priority == Log.ERROR && throwable != null){
```

Comment:
Use Log.e or Log.wtf otherwise

Reply:
@iamareebjamal Log.wtf has no int value to represent as Log.ERROR (=6). Please suggest how should I do that?